tools/js_manipulation.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# Copyright 2019 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+from . import shared
+
+
+def add_files_pre_js(user_pre_js, files_pre_js):
```

Comment:
Is the idea that this file with grow over time?  What kind of things go here?

Reply:
Perhaps the modularize and various other js code changes could happen here?